Sources/OktaLogger/Analytics/AppCenterAnalyticsProvider.swift

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2020-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import Foundation
+import AppCenterAnalytics
+
+public class AppCenterAnalyticsProvider: NSObject, AnalyticsProviderProtocol {
+
+    public var defaultProperties: [String: String]?
+    public var name: String
+    public var logger: OktaLoggerProtocol
+
+    private var provider: AppCenterAnalyticsProviderProtocol.Type
```

Comment:
can we please rename this variable and protocol to indicate that it is just a wrapper around epicenter analytics.
May be 
variable name:  `appCenter`
protocol name: `AppCenterProtocol`